ionic/src/app/add-employee/add-employee.component.ts

```diff
@@ -1,13 +1,16 @@
 import { Component, OnInit } from '@angular/core';
+import { DeviceService } from "../api/device.service"
 
 @Component({
   selector: 'app-add-employee',
   templateUrl: './add-employee.component.html',
   styleUrls: ['./add-employee.component.scss']
 })
 export class AddEmployeeComponent implements OnInit {
-
-  constructor() { }
+  devices: Device[] = [];
```

Comment:
i dont have employe id for particular employe thats  why i remove function from here for now